app/Http/Controllers/QuoteProposal/ReviewQuoteProposalcontroller.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers\QuoteProposal;
+
+use App\Constants\QuoteProposalTypeConstant;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\QuoteProposal\CreateQuoteProposalRequest;
+use App\Models\QuoteProposal;
+
+class ReviewQuoteProposalcontroller extends Controller
+{
+
+    public function __invoke(CreateQuoteProposalRequest $request, QuoteProposal $quote_proposal)
```

Comment:
```suggestion
    public function __invoke(CreateQuoteProposalRequest $request, QuoteProposal $quote_proposal): JsonResponse
```

You should always set an return type on functions.